vcd/resource_vcd_org_vdc.go

```diff
@@ -110,10 +110,11 @@ func resourceVcdOrgVdc() *schema.Resource {
 				Description: "True if this VDC is enabled for use by the organization VDCs. Default is true.",
 			},
 			"storage_profile": &schema.Schema{
-				Type:     schema.TypeList,
-				Required: true,
-				ForceNew: true,
-				MinItems: 1,
+				Type:        schema.TypeList,
```

Comment:
Just a thought here as I see it now. Is this not going to cause problems if/when storage profiles are not returned in the same order during READ?

Reply:
no complains until now, yet exists such possibility

Reply:
There is a [complaint](https://github.com/vmware/terraform-provider-vcd/issues/404)

Reply:
converted to typeSet